better-font-awesome.php

```diff
@@ -128,21 +128,21 @@ class Better_Font_Awesome_Plugin {
      *
      * @return  Better_Font_Awesome  The BFA object.
      */
-    public static function get_instance( $args = '' ) {
-        static $instance = null;
-        if ( null === $instance ) {
-            $instance = new static( $args );
-        }
+	public static function get_instance( $args = '' ) {
 
-        return $instance;
-    }
+		if ( ! self::$instance ) {
+			self::$instance = new static( $args );
```

Comment:
Your construct doesn't accept any arguments so no, don't accept args. 

Generally if the construct can accept args, predefine them in the get_instance method and allow the construct to be publicly accessible so that a developer can create their own singleton with their own custom args.